src/Data/Connection/PostObjectConnectionResolver.php

```diff
@@ -334,7 +334,7 @@ public function get_query_args() {
 						}
 					}
 
-					$query_args['orderby'][ esc_sql( $orderby_input['field'] ) ] = esc_sql( $order );
+					$query_args['orderby'][ $orderby_input['field'] ] = (string) $order;
```

Comment:
Not sure whether enum values need to be escaped, but for consistency we should do the same thing in other places: https://github.com/wp-graphql/wp-graphql/blob/8dc66601a825f3616d6701a5d875e0f509a15dbd/src/Data/Connection/PostObjectConnectionResolver.php#L324